servicex/servicex_remote.py

```diff
@@ -0,0 +1,183 @@
+import asyncio
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from typing import Optional, Tuple, List, Dict
+
+import aiohttp
+from minio import Minio, ResponseError
+from backoff import on_exception
+import backoff
+
+from .utils import ServiceXException, ServiceXUnknownRequestID
+
+
+# Low level routines for interacting with a ServiceX instance via the WebAPI
+
+async def _get_transform_status(client: aiohttp.ClientSession, endpoint: str,
```

Comment:
This would be tidier if it were a class. Session and endpoint (and eventually user credentials) will be properties

Reply:
Ok - I'll take a look!